ntbs-integration-tests/NtbsWebApplicationFactory.cs

```diff
@@ -20,6 +21,14 @@ public class NtbsWebApplicationFactory<TStartup> : WebApplicationFactory<TStartu
     {
         private string _testClassName;
 
+        protected override IHost CreateHost(IHostBuilder builder)
+        {
+            builder.UseSerilog();
+            var host = builder.Build();
+            host.Start();
+            return host;
+        }
+

```

Comment:
Another change to put `UseSerilog()` in the right place, but for the integration tests